backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/invoice/invoiceCandidateQtyToInvoiceOverride.feature

```diff
@@ -0,0 +1,1054 @@
+@from:cucumber
+Feature: Extend invoice-candidate test-coverage to IC QtyToInvoiceOverride
+
+  Background:
+    Given the existing user with login 'metasfresh' receives a random a API token for the existing role with name 'WebUI'
+    And set sys config boolean value true for sys config SKIP_WP_PROCESSOR_FOR_AUTOMATION
+    And set sys config boolean value false for sys config AUTO_SHIP_AND_INVOICE
+    And metasfresh has date and time 2021-04-16T13:30:13+01:00[Europe/Berlin]
+
+  @from:cucumber
+  @Id:S0156.2_100
+  Scenario:
+  In:
+  C_Order.IsSoTrx = Y
+  C_Order.QtyOrdered = 100 PCE / 25 kg
+  C_Invoice_Candidate.QtyDelivered = 100 PCE / 26,3547 kg
```

Comment:
in DE we have the decimal-`,`, but since this feature file is in EN, better use the decimal-`.` to avoid FUD